nemo/collections/asr/parts/numba/rnnt_loss/utils/cuda_utils/gpu_rnnt_kernel.py

```diff
@@ -397,3 +397,455 @@ def compute_grad_kernel(
             # update internal index through the thread_buffer;
             # until idx < V + 1, such that entire vocabulary has been updated.
             idx += GPU_RNNT_THREAD_SIZE
+
+
+@cuda.jit()
+def compute_multiblank_alphas_kernel(
+    acts: torch.Tensor,
+    denom: torch.Tensor,
+    sigma: float,
+    alphas: torch.Tensor,
+    llForward: torch.Tensor,
+    xlen: torch.Tensor,
+    ylen: torch.Tensor,
+    mlabels: torch.Tensor,
+    minibatch: int,
```

Comment:
Hmm you're right. I never noticed that... I was copying the function signatures of the original RNN-T functions in this file, and this variable was not used in those functions either.

@titu1994 shall we remove this variable from all function signatures in this file?

Reply:
Keep the arguments. I put them all there because i wanted the signatures to be similar for all kernels, and i expect future kernels may eventually need these parameters. In such a case, i want to avoid having different signature for different kernels v